### ISSUE TITLE
Duration affecting rest position

### ISSUE THREAD

**Issue by rettinghaus** [rism-ch/verovio#1920]
When a rest in layer gets shifted to avoid collision it seems to take its full duration into account instead of its visual width:  
![rest_position](https://user-images.githubusercontent.com/7693447/103537407-a24cca80-4e94-11eb-929a-bd873307f990.png)
```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title>Duration affecting rest position</title>
         </titleStmt>
         <pubStmt>
            <respStmt>
               <persName role="encoder">Klaus Rettinghaus</persName>
               <corpName role="funder">Enote GmbH</corpName>
            </respStmt>
            <date isodate="2021-01-04">4 Jan 2021</date>
         </pubStmt>
      </fileDesc>
   </meiHead>
   <music>
      <body>
         <mdiv>
            <score>
               <scoreDef>
                  <staffGrp>
                     <staffDef n="1" lines="5" clef.shape="G" clef.line="2" />
                  </staffGrp>
               </scoreDef>
               <section>
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <rest dur="16" />
                           <note dots="1" dur="8" oct="4" pname="e" accid.ges="n" />
                        </layer>
                        <layer n="2">
                           <rest dur="4" />
                        </layer>
                     </staff>
                  </measure>
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <rest dur="16" />
                           <beam>
                              <note dur="16" oct="4" pname="e" accid.ges="n" />
                              <note dur="8" oct="4" pname="c" accid.ges="n" />
                           </beam>
                        </layer>
                        <layer n="2">
                           <rest dur="4" />
                        </layer>
                     </staff>
                  </measure>
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <rest dur="16" />
                           <beam>
                              <note dur="16" oct="4" pname="e" accid.ges="n" />
                              <note dur="16" oct="4" pname="c" accid.ges="n" />
                              <note dur="16" oct="3" pname="a" accid.ges="n" />
                           </beam>
                        </layer>
                        <layer n="2">
                           <rest dur="4" />
                        </layer>
                     </staff>
                  </measure>
                  <measure>
                     <staff n="1">
                        <layer n="1">
                           <rest dur="16" />
                           <note dur="16" oct="4" pname="e" accid.ges="n" />
                           <note dur="16" oct="4" pname="c" accid.ges="n" />
                           <note dur="16" oct="3" pname="a" accid.ges="n" />
                        </layer>
                        <layer n="2">
                           <rest dur="4" />
                        </layer>
                     </staff>
                  </measure>
                  <measure>
                    <staff n="1">
                       <layer n="1">
                          <rest dur="16" />
                          <note dur="16" oct="4" pname="e" accid.ges="n" />
                          <note dur="16" oct="3" pname="a" accid.ges="n" />
                          <note dur="16" oct="3" pname="d" accid.ges="n" />
                       </layer>
                       <layer n="2">
                          <rest dur="4" />
                       </layer>
                    </staff>
                 </measure>
              </section>
            </score>
         </mdiv>
      </body>
   </music>
</mei>
```

**Comment by rettinghaus** [comment]
This has been fixed.

**Comment by lpugin** [comment]
This what I get with 3.2.0-dev-ec3a48e

![image](https://user-images.githubusercontent.com/689412/105990883-65df4980-60a3-11eb-8a3c-6ed524c6b75a.png)

**Comment by rettinghaus** [comment]
We discussed this thoroughly and decided that basic behaviour is correct in principle, but rests should not automatically be moved too far away from the staff. Fixed in #1950
